vignettes/Intro.Rmd

```diff
@@ -73,6 +73,21 @@ log_eval(y <- f(g(x)), level = INFO)
 str(y)
 ```
 
+
+Sometimes, it may be reasonable to want to log a smallish data.frame or data.table, e.g. mtcars.
+
+```{r}
+formatter_data_frame <- function(df, ...) {
+    pander::pander_return(df, style = 'simple')
+}
+log_formatter(formatter_data_frame)
+
+data(mtcars)
+log_info(mtcars)
+
+```
```

Comment:
```suggestion
Sometimes, it may be reasonable to log R objects as markdown, e.g. a smallish `data.frame` or `data.table`, e.g. `mtcars` or `iris`. Calling the formatter using `pander` instead of `glue` can help:

```{r}
pander::panderOptions('table.style', 'simple')
log_formatter(formatter_pander)
log_info(head(iris))
```
```